salt/modules/state.py

```diff
@@ -41,6 +41,7 @@
 from salt.exceptions import CommandExecutionError, SaltInvocationError
 from salt.runners.state import orchestrate as _orchestrate
 from salt.utils.odict import OrderedDict
+from salt.utils import to_unicode
```

Comment:
`salt.utils` is deprecated, this would need to be imported from `salt.utils.stringutils`.